### ISSUE TITLE
Needs an update to support Broccoli >= 1.0

### ISSUE THREAD

**Issue by adambullmer** [stfsy/broccoli-inject-livereload#4]
```Warning: InjectLivereload: The .read/.rebuild API is no longer supported as of Broccoli 1.0. Plugins must now derive from broccoli-plugin. https://github.com/broccolijs/broccoli/blob/master/docs/broccoli-1-0-plugin-api.md```

Looks like this project needs a quick update to be based off the broccoli-plugin base class.

**Comment by Dremora** [comment]
Unfortunately, it's time to admin I can't maintain this library as I don't use Broccoli anymore. However I'll be happy to transform the ownership (along with publishing rights on npm) if you are willing to solve this problem yourself (and maybe even deal with a long-standing issue #3).

**Comment by stfsy** [comment]
hey @Dremora i'm mainting a broccoli livereload plugin, too. If you're still willing to transfer the ownership, you can transfer it to me. I'd merge both plugins in the coming weeks.

**Comment by Dremora** [comment]
@stfsy sorry, only read this now. I've initiated the transfer, if you still want to do it.

**Comment by stfsy** [comment]
Yup, sure

**Comment by Dremora** [comment]
You now also have permissions to publish the package on npm.

**Comment by stfsy** [comment]
allright @adambullmer i just published v**1.0.0** to npm.

 New Implementation 
- injects the livereload script only in html files that match the **target** option and
- expects the port to be nested in a **livereload** object. 
```js
const BroccoliInjectLivereload = require('broccoli-inject-livereload')

const reloadable = new BroccoliInjectLivereload('app', {
    target: 'index.html',
    livereload: {
        port: 12345
       }
    })

module.exports = reloadable
```
